### DISCUSSION TITLE
Shorter chapters

### DISCUSSION THREAD

**Discussion by cgillinger** [gopro/labs#373] [Feature Requests]
Hi! I found the larger chapter support (eg https://gopro.github.io/labs/control/chapters/), but I'm looking for *shorter* chapter support.

My user case is as follows: I use my GoPro Max to make Streetview compliant videos, eg to make "Street View blue lines" on Google Maps. The StreetView studio (https://streetviewstudio.maps.google.com/) is very sensitive to GPS-data losses and I get them a lot on my Gopro Max. One GPS gap means the whole video is rejected when uploading to Streetview Studio.

So, what I'm looking for is a way to set the chapters for a fixed time. Say each chapter is max 2-3 minutes. This would mean that a loss of GPS data would not impact the whole video, but just that chapter and with sufficient short chapters, this would mean a lot more of the video would be usable.

Is this possible?

**Comment by dnewman-gpsw** [comment]
Not directly on Max.  It is offered on HERO11, where you can set your chapter size from 100MB to 64GB. 
For MAX there are is one way to short their length a little, by increasing your bit-rate.  The default is 60Mb/s, but Max Labs supports up to 100Mb/s (90Mb/s is reliable for most cameras.)   Max has 9 minute chapters, at 90mb/s they become 6 minutes.

**Reply by cgillinger** [reply]
Ah, bummer. Not sure if even six minutes is too long, but I'll give it a try. I also tried using the "start on speed", eg recording starts att speed x and stops when it drops below speed y, but everytime I get a gps drop, the speed is registered as "0" and recording stops automatically.

**Reply by dnewman-gpsw** [reply]
Isn't that the best solution, record when GPS is good?  The new Max Labs release support a trigger just based on DOP (GPS precision), so your records when only have valid GPS data.

**Reply by cgillinger** [reply]
Thats what I thought as well. However the amount of small glitches makes the file number go up, and it takes quite some time for it to restart recording again. So I ended up with a gazillion short files, with lots of gaps between them.

With short chapters, I get a bit more files, and I will loose some of them, but it would have been more manageable. I think.

**Reply by dnewman-gpsw** [reply]
So you tried the DOP trigger in scripting? It is not the same a the speed trigger.  Plus you could even using the scripting to simply create short chapters.  Any coding experience?
